zerver/lib/push_notifications.py

```diff
@@ -668,27 +667,25 @@ def get_gcm_alert(
         and trigger == NotificationTriggers.PRIVATE_MESSAGE
     ):
         return f"New direct message from {sender_str}"
-    elif message.is_stream_message() and trigger == NotificationTriggers.MENTION:
+
+    assert message.is_stream_message()
```

Comment:
So this change has an implicit assumption that it won't be possible to disable push notifications for private messages by default but then still have some feature, like a mention or an alert word, that can trigger a push notification if THAT appears in a private message.

We don't anything of that form today, but I guess if we added such a thing, it would belong in the above block anyway, and would want new rendering logic, so this is OK. @prakhar1144 FYI.

Reply:
Yup, I think we could cross that bridge if we got there.  Ideally it will always be fairly simple to triage stream-related stuff from DM-related stuff